src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const Price = 40;
+  const PriceForWeek = 50;
+  const PriceForThreeDays = 20;
+
+  if (days >= 7) {
+    return days * Price - PriceForWeek;
+  }
+
+  if (days >= 3) {
+    return days * Price - PriceForThreeDays;
```

Comment:
Similarly, use a constant to represent the minimum number of days that qualify for a short-term rental discount instead of the literal `3`.